ir/src/main/scala/org/scalajs/core/ir/Printers.scala

```diff
@@ -681,10 +681,19 @@ object Printers {
 
         case IntLiteral(value) =>
           if (value >= 0) {
-            print(value)
+            print(value.toString)
           } else {
             print('(')
-            print(value)
+            print(value.toString)
+            print(')')
+          }
+
+        case LongLiteral(value) =>
+          if (value >= 0) {
+            print(value.toString)
+          } else {
+            print('(')
+            print(value.toString)
             print(')')
```

Comment:
In that case I would assume that we would also add them to floats and doubles. I believe that a change of representation like this should be in another commit.